WordPress/Classes/Models/Blog+History.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+extension Blog {
+
+    /// Returns the blog currently flagged as the one last used, or the primary blog,
+    /// or the first blog in an alphanumerically sorted list, whichever is found first.
+    @objc(lastUsedOrFirstBlogInContext:)
+    static func lastUsedOrFirstBlog(in context: NSManagedObjectContext) -> Blog? {
```

Comment:
I didn't think this while reading the method name, but, after seeing this in action, I wonder if we can do without the "Blog". 

Without it, the usage would be:

```swift
Blog.lastUsedOrFirst(in: context)
```

That reads fine to me, and it seems pretty clear that the return value would be a `Blog`.

What do you think? I'm not too bothered by it, just an idea. But, if you like the suggestion, I think we should document it somewhere as the desired style for these ActiveRecord-like methods.

Reply:
Good point. The "Blog" does seem redundant. 👍